outline.md

```diff
@@ -0,0 +1,90 @@
+
+## Learning Path
+
+* The abstract said this was an Oriole.  It's actually a Learning Path.  Both have text and video components.
+* Modules of information about inner source.
+* Modules are independently consumable or as a group.
+* No assumption of how the content of the modules will be applied or extended.
+* Content consumable via video _or_ text.
+
+## The Current Content
+
+* Presents inner source as two teams with one dependent on the other.
+* Inner source comes into play when the dependent team can't get a requested feature.
+* We review the drawbacks of wait it out, work around, and escalate.
+* An inner source contribution is defined in terms of:
+  * **Host** and **guest** teams
+  * A **contributor** submitting code.  A **trusted committer** reviewing the fitness code and mentoring the contributor.  And a **product owner** reviewing the desirability of the requested feature.
+* The benefits of inner source are listed as:
+  * A company-scalable way for guests to get in their features.
+  * Expanded engineering ability for hosts with high demand for features.
+  * Improved requirements and prioritization and alignment
+  * Break down company engineering silos
+* Specific shapes and setup of inner source may vary.  For that reason the training also lays down core, unvarying principles that all implementations of inner source should follow to be successful in producing those benefits and solving that problem.
+  * Voluntary code contribution.  On the part of the guests and hosts.
+  * Prioritized mentorship.  From the host to the guest.
+  
+## How to Access
+
+* The learning path will be available in _Safari_.  Everyone here gets a free _Safari_ membership.
+* We are still in post-production on the specific learning path.  Leave your favorite emoji on my recent post in the _innersourcecommons_ _Slack_ and I will email you when the content is finished.
+* We are also making the videos and text freely-available outside of _Safari_.
+
+## What's next
+
+* There is more training coming.  The same modular, multi-media approach to habits and patterns for success as a contributor, trusted committer, and product owner involved with inner source.
+* There is opportunity to get involved in the production of those future modules.  Lots of ways to be involved:
+  * Speaker (speaking evaluator).  Do the speaking content.  Probably about 20 hours of time.  (2 hours as a review)
+  * Writer (copy reviewer).  Produce the written content.  Probably about 40 hours of time.  (4 hours as a review)
+  * Post-production (also reviewer).  Makin' slides, etc.  Probably about 10 hours of time.  (1 hour as a review)
+* Share my own experience on being involved in the process.
+
+## Call to action
+
+* Watch and read the content.
+* Share it with others.
+* Use the terms in your conversation with others.  Doing so should result in conversations and writing that we can all understand more-quick and share and apply more-broadly.
```

Comment:
Use the diagrams when sharing terms.  This will ensure that the story is told with less brand dilution and bias.